### ISSUE TITLE
[Android 10] Notification seek bar does not reinitiate to 0 when automatically seeking to next track

### ISSUE THREAD

**Issue by ainar** [react-native-kit/react-native-track-player#920]
### The bug
When a track ends and the player automatically skips to the next track, the seek bar in the notification is not resetting to the beginning and continues its progress from the end time of the last track (or get stick at the end of the next track if it is shorter).

_The track is playing normally, and all the methods are returning the correct progress (the `Progressbar` component is working). The seek function of the seek bar in notifications is also working._

### How to reproduce
**With Android 10 only.** Behavior reproducible in the example app:
1. in a queue with several tracks, listen to a track until the end (or seek to the end if you are impatient and you set the appropriate capability)
2. notice the progress of the seek bar during the next track

### Environment
`react-native info`

```
System:
    OS: Windows 10 10.0.18363
    CPU: (4) x64 Intel(R) Core(TM) i5-2500 CPU @ 3.30GHz
    Memory: 6.25 GB / 15.98 GB
  Binaries:
    Node: 12.16.1 - C:\Program Files\nodejs\node.EXE
    Yarn: 1.22.4 - C:\Program Files (x86)\Yarn\bin\yarn.CMD
    npm: 6.13.4 - C:\Program Files\nodejs\npm.CMD
    Watchman: Not Found
  SDKs:
    Android SDK: Not Found
  IDEs:
    Android Studio: Version  3.6.0.0 AI-192.7142.36.36.6392135
  Languages:
    Java: javac 14 - C:\Program Files\Java\jdk-14\bin\javac.EXE
    Python: 3.7.7 - C:\ProgramData\Anaconda3\python.EXE
  npmPackages:
    @react-native-community/cli: Not Found
    react: 16.11.0 => 16.11.0
    react-native: 0.62.2 => 0.62.2
  npmGlobalPackages:
    *react-native*: Not Found
```

_What `react-native-track-player` version are you using?_
**1.2.3**

_Are you testing on a real device or in the simulator? Which OS version are you running?_
Real device: **Samsung Galaxy S9 with Android 10**
Simulator: Pixel 2 with Android 9 (but there is no notification seek bar in Android 9 so we cannot see the problem)

**Code**
[Example.](https://github.com/react-native-kit/react-native-track-player/tree/dev/example)

**Comment by lnickdnguyen** [comment]
Noticed the same thing^

**Comment by Guichaguri** [comment]
Let me know if https://github.com/react-native-kit/react-native-track-player/commit/bb7c183d2ff217ec70f63a95b7da8105f34911b2 fixes it for you.

**Comment by curiousdustin** [comment]
Fix released in [1.2.4](https://github.com/react-native-kit/react-native-track-player/releases/tag/v1.2.4)
